app/App.tsx

```diff
@@ -12,16 +12,20 @@ enum PageDisplay {
 
 function App(): ReactElement {
   const [displayPage, setdisplayPage] = useState(PageDisplay.Login);
+  const [displayAccountCreatedMessage, setDisplayAccountCreatedMessage] = useState(false);
   return (
     <AppStyling>
       {displayPage === PageDisplay.Login && (
         <Login
           onLoginSuccess={() => {
             setdisplayPage(PageDisplay.Welcome);
+            setDisplayAccountCreatedMessage(false)
           }}
           onAccountCreationClick={() => {
             setdisplayPage(PageDisplay.AccountCreation);
+            setDisplayAccountCreatedMessage(false)
           }}
+onAccountCreationSuccess={displayAccountCreatedMessage===true?"Account created":null}
```

Comment:
`onAccountCreationSuccess` sounds like it's expecting a function, not a string.

in the ternary, you have  `displayAccountCreatedMessage===true`, is there any way to simplify this?

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/10/commits/bc86edd35f30606339369565ad37ae4d78e0d787